src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -0,0 +1,48 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.mapper.impl.response.OrderResponseMapper;
+import mate.academy.spring.model.Order;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.response.OrderResponseDto;
+import mate.academy.spring.service.OrderService;
+import mate.academy.spring.service.ShoppingCartService;
+import mate.academy.spring.service.UserService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+    private final OrderService orderService;
+    private final ShoppingCartService shoppingCartService;
+    private final UserService userService;
+    private final OrderResponseMapper orderResponseMapper;
```

Comment:
```suggestion
    private final DtoResponseMapper<OrderResponseDto, Order> orderResponseMapper;
```

Reply:
![image](https://user-images.githubusercontent.com/81781479/183257368-b14b8224-31c6-4dd0-a16a-9a6b8a9adead.png)
Before you create PR with your homework, you need to go through the description of the task and checklist under the task, and correct all the points described there. The mentor will not check the work until the checklist items are corrected